libethcore/Farm.h

```diff
@@ -48,6 +48,16 @@ namespace dev
 {
 namespace eth
 {
+struct FarmSettingsType
```

Comment:
Because it is a typedef. It's a personal habit (which helps to differentiate data types from variable pointers).

I personally find more intuitive to read

```
FarmSettingsType FarmSettings;
```
instead of (for example)
```
FarmSettings farmSettings;
```

Reply:
I don't agree with this reasoning. I don't see a point in marking types with `Type` suffix. Please, let's stick more/less to the naming convention we currently have.

Reply:
I will comply nevertheless I underline it does not make any harm.

Reply:
I'm saying I hate it. I only try to make the naming convention consistent to make it a bit easier for other contirbutors.

Reply:
> I'm saying I hate it.

Isn't a bit rude ?
If there are other conventions/techniques which are so horrible in my coding please let me know.

Reply:
Should I also amend `Enum` suffixes ?

https://github.com/ethereum-mining/ethminer/blob/master/libethcore/Miner.h#L42_L48

Reply:
No they're not in this PR nevertheless they're still from my previous PR #1751

Reply:
>  they're still from my previous PR #1751

That's ok.

Reply:
Remove "Type" suffix